src/backend/distributed/shared_library_init.c

```diff
@@ -1272,6 +1273,18 @@ RegisterCitusConfigVariables(void)
 		GUC_NO_SHOW_ALL,
 		NULL, NULL, NULL);
 
+	DefineCustomBoolVariable(
+		"citus.hide_citus_dependent_objects",
+		gettext_noop(
+			"Hides some objects, which depends on citus extension, from pg meta class queries."
+			"It is intended to be used only before postgres vanilla tests to not break them."),
+		NULL,
+		&HideCitusDependentObjects,
+		false,
+		PGC_USERSET,
```

Comment:
can we use `PGC_POSTMASTER` instead of `PGC_USERSET`? In the end, we only set this at the start of regression tests by changing the postgres config, and never change?

But please try before changing this.